test/helpers/sockets.rb

```diff
@@ -0,0 +1,435 @@
+# frozen_string_literal: true
+
+require 'socket'
+require 'timeout'
+
+module TestPuma
```

Comment:
I'd prefer `Puma::Test` unless you have a namespace-lookup related reason to do this

Reply:
Actually, it's both a namespace issue and a doc issue.  If we name it Puma::Test, all the test items will appear under `Puma` (class/module names will be shown as constants, etc).

`TestPuma` puts them outside of Puma, and an alpha list of classes (see https://msp-greg.github.io/puma/) puts all the test classes/modules below everything in the current list...